client/app-bar/lockup.jsx

```diff
@@ -23,15 +31,22 @@ const SizedLogoText = styled(LogoText)`
   margin-left: 8px;
 `
 
+const VersionText = styled(Caption)`
+  margin: -15px 0px;
```

Comment:
Per material design spec, try to stick to using a number divisible by 4 when using margins/paddings, unless there's a good reason to do otherwise. `-16px` would probably be a better choice here.